src/views/document/utils/boxes/ElevationProfile.vue

```diff
@@ -95,12 +96,16 @@ export default {
     if (this.hasData) {
       d3.then(this.createChart);
     }
+
+    this.debouncedOnWindowResize = debounce(this.onWindowResize, 300);
```

Comment:
I'd rather use a resizeObserver on the graph element for performance reasons, because it's size may not always change when the window is resized.

Reply:
ok done